assets/texts/es/amoSummary.txt

```diff
@@ -1,3 +1,3 @@
-Esta extensión te permite generar rápidamente códigos QR offline de una URL abierta en tus pestañas o cualquier texto (seleccionado)! 👍
+Esta extensión te permite generar rápidamente códigos QR de la URL de la pestaña abierta o cualquier otro texto! 👍
```

Comment:
IMHO it would be clearer as follows:

```suggestion
Esta extensión te permite generar rápidamente un código QR con la URL de la pestaña actual, ¡o con cualquier otro texto! 👍
```

By the way, although I know that in texting/chat we typically omit them, in longer-form text [_Ortografía de los signos de interrogación y exclamación_](https://www.rae.es/espanol-al-dia/ortografia-de-los-signos-de-interrogacion-y-exclamacion) still applies.